src/webapp/components/projects/ProjectTable.tsx

```diff
@@ -126,16 +134,28 @@ export const ProjectTable = React.memo(() => {
                                 });
                             },
                             error => {
-                                loading.hide();
+                                setLoading(false);
                                 snackbar.error(error.message);
                                 return reject(new Error(error.message));
                             }
                         );
                 });
             },
-            [compositionRoot.projects.get, loading, snackbar]
+            [compositionRoot.projects.get, snackbar, refreshTable]
         )
     );
 
-    return <ObjectsTable {...tableConfig} />;
+    const reloadProjects = React.useCallback((isCancelAction: boolean) => {
```

Comment:
minor, no need to change it, but as a general rule, naked boolean arguments are not declarative when used: `reloadProjects(false)`. Instead: `reloadProjects({ isCancelAction: false })`